tf2_py/test/test_buffer_core.py

```diff
@@ -0,0 +1,261 @@
+# ***********************************************************
+# * Software License Agreement (BSD License)
+# *
+# *  Copyright (c) 2009, Willow Garage, Inc.
```

Comment:
@tfoote should new files be copyright `Open Source Robotics Foundation, Inc.`? Also, 2019 instead of 2009.